example/components/ConnectWithSelect.tsx

```diff
@@ -26,14 +26,12 @@ function ChainSelect({
       }}
       disabled={switchChain === undefined}
     >
-      <option hidden disabled selected={activeChainId === undefined}>
+      <option hidden disabled>
         Select chain
       </option>
-      <option value={-1} selected={activeChainId === -1}>
-        Default
-      </option>
+      <option value={-1}>Default</option>
       {chainIds.map((chainId) => (
-        <option key={chainId} value={chainId} selected={chainId === activeChainId}>
+        <option key={chainId} value={chainId}>
           {CHAINS[chainId]?.name ?? chainId}
         </option>
```

Comment:
React was complaining about this invalid syntax in the example, but it's not related to the core diff here.